bucket/chatterino.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "2.1.5-4",
+    "homepage": "https://chatterino.com",
+    "description": "Chat client for twitch chat",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://chatterino.com/download/2.1.5-4/Chatterino%20Portable.zip",
+            "hash": "b3326f45f7d31f5f0b767b4805732f5e8192d0a141d61ccd86b367d41edb32f0"
+        }
+    },
+    "extract_dir": "chatterino2",
+    "bin": "chatterino.exe",
+    "shortcuts": [
+        [
+            "chatterino.exe",
+            "Chatterino"
+        ]
+    ],
+    "persist": [
+        "Settings",
+        "Cache"
+    ],
+    "suggest": {
```

Comment:
Move suggest before `architecture` property

Reply:
Included change in a single commit with another change. (Change of description)

Reply:
Added to a commit.